framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -202,26 +202,20 @@ public boolean hasDefaultsForCheckedCode() {
         return false;
     }
 
-    /**
-     * Add standard unchecked defaults that do not conflict with previously added defaults.
-     *
-     * @param tops AnnotationMirrors that are top
-     * @param bottoms AnnotationMirrors that are bottom
-     */
-    public void addUncheckedStandardDefaults(
-            Iterable<? extends AnnotationMirror> tops,
-            Iterable<? extends AnnotationMirror> bottoms) {
+    /** Add standard unchecked defaults that do not conflict with previously added defaults. */
+    public void addUncheckedStandardDefaults() {
         for (TypeUseLocation loc : standardUncheckedDefaultsTop) {
             // Only add standard defaults in locations where a default has not be specified
-            for (AnnotationMirror top : tops) {
+            for (AnnotationMirror top : atypeFactory.getQualifierHierarchy().getTopAnnotations()) {
```

Comment:
Move the `getTop/BottomAnnotations` calls outside of the for loops, like you do in `addClimbStandardDefaults`.

Reply:
Done.